src/EventHub.Application.Contracts/EventHubApplicationContractsModule.cs

```diff
@@ -11,6 +12,9 @@ namespace EventHub
     )]
     public class EventHubApplicationContractsModule : AbpModule
     {
-
+        public override void ConfigureServices(ServiceConfigurationContext context)
+        {
+            LimitedResultRequestDto.MaxMaxResultCount = 30;
```

Comment:
Why we needed that? You can add a comment to the code for the next developer.

Reply:
I did this to protect ourselves. Currently, requests can be made to the API from places such as Postman. I think this is a necessary limitation for users making requests to the API.

<img width="1680" alt="Screen Shot 2021-12-24 at 13 25 53" src="https://user-images.githubusercontent.com/31216880/147357122-9e61f0b3-285c-4a17-ab96-0eb6cdc0f05e.png">